src/Client/Pages/BuildingBlock/SearchComponent.fs

```diff
@@ -185,15 +188,15 @@ let private AddBuildingBlockButton (model: Model) inputValue dispatch =
 [<ReactComponent>]
 let Main (model: Model) dispatch =
     let state_bb, setState_bb = React.useState(BuildingBlockUIState.init)
-    let inputValue, setInputValue = React.useState ""
+    let commentHeader, setCommentHeader = React.useState model.AddBuildingBlockState.CommentHeader
```

Comment:
This is really bad style!! Just use the value straight from model. No sense in binding it one more time to a react state